packages/block-library/src/table/edit.js

```diff
@@ -7,7 +7,15 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { Fragment, Component } from '@wordpress/element';
```

Comment:
Nitpick: it would be nice if imports were ordered alphabetically. They tend to be in a random order that varies from file to file currently.

Reply:
Seems a bit unnecessary when there are just two imports.